registry/api/v2/urls.go

```diff
@@ -220,15 +220,3 @@ func appendValuesURL(u *url.URL, values ...url.Values) *url.URL {
 	u.RawQuery = merged.Encode()
 	return u
 }
-
-// appendValues appends the parameters to the url. Panics if the string is not
-// a url.
-func appendValues(u string, values ...url.Values) string {
-	up, err := url.Parse(u)
-
-	if err != nil {
-		panic(err) // should never happen
```

Comment:
@levigross I understood that. I was responding to your assertion in the PR description. I am not sure why that is. I suspect I mean to export this for use elsewhere, but we can still remove it.